contracts/0.8.25/vaults/predeposit_guarantee/PredepositGuarantee.sol

```diff
@@ -404,117 +410,87 @@ contract PredepositGuarantee is IPredepositGuarantee, CLProofVerifier, PausableU
 
         if (unlockedGuarantee < totalDepositAmount) revert NotEnoughUnlocked(unlockedGuarantee, totalDepositAmount);
 
-        // staging 31 ETH to be able to activate the validator as it gets proved
```

Comment:
maybe worth checking available balance explicitly to fail before spending the gas on deposit